programs/svm-spoke/src/state/instruction_params.rs

```diff
@@ -0,0 +1,12 @@
+use anchor_lang::prelude::*;
+
+use crate::RelayerRefundLeaf;
+
+#[account]
+#[derive(InitSpace)]
+pub struct ExecuteRelayerRefundLeafParams {
+    pub root_bundle_id: u32,
+    pub relayer_refund_leaf: RelayerRefundLeaf,
+    #[max_len(0)]
```

Comment:
Are they actually empty when we pre-load the instruction params account?

Reply:
This only is helper to calculate total account space when initializing. Initially its all zeroed, but it is up to the user to correctly encode raw data when writing fragments to this account.